syntax.d/cpp.ini

```diff
@@ -0,0 +1,9 @@
+name=C++
+extensions=cpp, hpp, cc, cxx, hxx
+highlight_numbers=true
+singleline_string_quotes="
+singleline_comment_start=//
+multiline_comment_delims=/*, */
+multiline_string_delim="
```

Comment:
These are single line string delimiters, not multiline. If a language doesn't have multi-line strings, you can just remove the line from the config. singleline_string_quotes and multiline_string_delim should not be equal

Reply:
Is `"""` actually valid C++? I thought only raw string could represent multi-line string.
Please also remove all the `multiline_string_delim` in other files that have the same value as `singleline_string_quotes`